src/NuGet.Core/NuGet.Packaging/Signing/Signature.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Packaging.Signing
+{
+    /// <summary>
+    /// Package signature information.
+    /// </summary>
+    public class Signature
```

Comment:
Should this type include the `X509Certificate2` used to create the signature? We will need this server-side to do validations.

Reply:
Is the cert going to be included in the package? This is the right class for including it once we know what the package/signing info looks like.

Reply:
Yup, the signing certificate will be included (but not the whole certificate chain)

Reply:
I'll add an API to get the full chain, this will be needed for verify also